english/coffee/nude-run/warm-water/warm-water.md

```diff
@@ -6,3 +6,12 @@ at last speaks again.
 
 "You know," says the badger, "the more I think about it, the more certain
 I become that we are all made of bees."
+
+He says this with the absurd certainty of Yoda, or the Little Prince. Part of
+you wants to scream (there is no such thing as speaking badgers, and I am
+certainly not made of bees). But there is also a small part of you that feels
+the pull of the badgers soft voice, his calm but engaging eyes.
+
+Do you:
+* [rebel against the absurdity of it all?] (rebel/rebel.md)
+* [let your mind surrender] (surrender/surrender.md)
```

Comment:
I like the idea of putting the links in a list, but I want to make it as easy as possible for people who don't know markdown to contribute to the repository. To make that easier, could you put the different options in different paragraphs instead of in a list? (That is, remove the asterisks and add a blank line between options.)